Mathlib/LinearAlgebra/TensorProduct/LinearMap.lean

```diff
@@ -0,0 +1,86 @@
+/-
+Copyright (c) 2024 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import Mathlib.LinearAlgebra.Dual
+import Mathlib.LinearAlgebra.TensorProduct.Tower
+
+/-!
+# Bilinear maps on a Tensor Product
+
+## Main definitions
+
+* `LinearMap.tensorDistrib (B₁ ⊗ₜ B₂)`: the bilinear map on `M₁ ⊗ M₂` constructed by applying
+  `B₁` on `M₁` and `B₂` on `M₂`.
+
+## Implementation notes
+
+Bilinear map versions of some results in `LinearAlgebra.BilinearForm.TensorProduct`.
```

Comment:
If we're claiming that `BilinForm` is the soon-to-be-deprecated version, then I think this comment belongs in `LinearAlgebra.BilinearForm.TensorProduct` pointing in the other direction.

Reply:
Done.